Atomic/util.py

```diff
@@ -245,11 +264,23 @@ def skopeo_layers(image, args=[], layers=[]):
 
 
 def check_v1_registry(image):
-    # Skopeo cannot interact with a v1 registry
-    netloc = (urlparse(image)).netloc
-    v1_url = "https://{}/v1/_ping".format(netloc)
-    if requests.get(v1_url).status_code == 200:
-        raise ValueError("\nUnable to interact with a V1 registry.")
+    """
+    Skopeo does not support v1 registry API.
+    See https://docs.docker.com/registry/spec/api/ API Version Check
+
+    :param image:
+    :return: Raise exception unless we can confirm v2 registry access
+    """
+    registry = list(urlparse(image))
+    registry[0] = "https"
+    registry[2] = "/v2/"
+    try:
+        if requests.get(urlunparse(registry)).status_code == 200:
+            pass
+    except requests.exceptions.SSLError:
+        registry[0] = "http"
```

Comment:
I'm not sure about automatically falling back to `http`.  Shouldn't we have some sort of per-registry flag like docker has?  I personally like the "remote" model like git and ostree have...might be interesting to try doing that here for docker images.